datafusion/physical-expr-functions-aggregate/src/aggregate.rs

```diff
@@ -61,11 +52,10 @@ pub mod utils;
 ///
 /// You can also create expression by [`AggregateExprBuilder`]
 #[allow(clippy::too_many_arguments)]
+#[deprecated(since = "40.0.0", note = "Use `AggregateExprBuilder` instead")]
 pub fn create_aggregate_expr(
```

Comment:
Given we are changing APIs, do we still need to keep it deprecated or just remove it at all?

Reply:
I think we should